catalog.bom

```diff
@@ -85,8 +147,35 @@ brooklyn.catalog:
           is.root.node: true
           launch.latch: $brooklyn:component("my-hyperledger-membersrvc-node").attributeWhenReady("service.isUp")
 
+  - id: hyperledger-fabric-cluster-single-location
+    item:
+      name: Hyperledger Fabric Cluster (Single-location)
+      type: org.apache.brooklyn.entity.stock.BasicApplication
+
+      brooklyn.children:
+      - type: org.apache.brooklyn.entity.group.DynamicRegionsFabric
```

Comment:
Suggest each entity propagates the appropriate value to a custom sensor and having line 369 consume the propagated value then.